### PR TITLE
Add "Termination of Services" section to ToS

### PR DESCRIPTION
**Pull request by vikaspotluri123**
As we plan on removing inactive accounts in the future, we'd like to explicitly explain how users can terminate or have their services terminated

### CODE REVIEW

**Review by ramsays — DISMISSED** [review]
Sounds good to me. Do we want to explicitly say/imply how long we will wait before considering an account "inactive"?

**Comment by vikaspotluri123** [comment]
I don't think so. The first deletion batch is going to have a much shorter turnaround time than normal, and we might redefine normal based on internal metrics.

**Comment by vikaspotluri123** [comment]
How's that?

**Comment by ramsays** [comment]
Works for me!

**Review by ramsays — APPROVED** [review]

File: tos.md [hunk 1]
```diff
@@ -36,6 +36,20 @@ Permission is granted to use the services and functionality on Gradebook's websi
...
+
+Gradebook reserves the right to terminate your account under certain conditions, including but not limited to:
+
+ - Account inactivity (though you will be given reasonable notice)
+ 
+ - Abuse
+ 
+ - Requesting account deletion through the Account page of the Application
+ 
+ - Requesting account deletion via email
```
Comment (joshcosta) [inline tos.md:49]:
```suggestion
 - Requesting account deletion via email to contact@gradebook.app
```

**Review by joshcosta — CHANGES_REQUESTED** [review]

File: tos.md (hunk 1 above)
Comment (vikaspotluri123) [reply tos.md:49]: I think leaving it at as-is is fine. Wdyt @ramsays?

Comment (ramsays) [reply tos.md:49]: I would say leave as-is.

**Review by joshcosta — APPROVED** [review]

**Pull request merged**